pkg/api/platformapi/allocatorapi/vacate.go

```diff
@@ -119,7 +121,7 @@ func moveAllocators(params *VacateParams, p *pool.Pool) ([]pool.Validator, bool,
 
 // moveNodes moves all of the nodes off the specified allocator
 func moveNodes(id string, params *VacateParams, p *pool.Pool) ([]pool.Validator, bool, error) {
-	var merr = multierror.NewPrefixed(fmt.Sprintf("allocator %s", id))
+	var merr = multierror.NewPrefixed("vacate error")
```

Comment:
@Z3r0Sum does this break our error parsing in soteria?

Reply:
This shouldn't break parsing; however, we'd want to update our tests to accurately reflect this change.